### PR TITLE
Implement prev/next buttons

### PR DESCRIPTION
**Pull request by Toxiapo**
This PR adds the Next and Previous buttons to the page.

Couple of things that we still need to take care of:

- On `Scroll` mode, the buttons should be hidden. Do we add an additional prop to the button or this could be handled in the parent component?
- Still need to fix the color/background when we introduce color mode.

### CODE REVIEW

**Comment by kristojorg** [comment]
I think for now let's assume the parent component just won't render the prev/next at all in scroll mode. 

We can also come back to color mode all at once in that color mode ticket. I'd like to see if we can use the built in chakra support for color modes while adding the third sepia option

File: src/ui/PageButton.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        alignItems="center"
+        border="1px solid"
+        borderColor="ui.gray.light-cool"
+        color="ui.black"
+        bg="inherit"
+        size={{ sm: '40px', md: '50px' }}
+        mr={8}
+        ml={6}
+        cursor="unset"
+        _hover={{ background: 'ui.black', color: 'ui.white' }}
```
Comment (hokei) [inline src/ui/PageButton.tsx]: I think the color on the reader is like, #4d4d4d.  Do we have a dark grey preset?

**Review by hokei — CHANGES_REQUESTED** [review]
Just one nit, and the lint warnings, which should be easy enough to remove and will make things cleaner.

File: src/ui/PageButton.tsx (hunk 1 above)
Comment (Toxiapo) [reply src/ui/PageButton.tsx]: Yes, we do! Thank you.

**Review by hokei — APPROVED** [review]

**Pull request merged**